extensions/wikia/AdEngine/js/AdContext.js

```diff
@@ -3,7 +3,11 @@
  * The AMD module to hold all the context needed for the client-side scripts to run.
  */
 define('ext.wikia.adEngine.adContext', [
-	'wikia.window', 'wikia.document', 'wikia.geo', require.optional('wikia.instantGlobals'),  require.optional('wikia.abTest')
+	'wikia.window',
+	'wikia.document',
+	'wikia.geo',
+	'wikia.instantGlobals',
```

Comment:
Seems like it's not failing after moving js module to the package common for both mercury and wikiamobile.